mrs.developer.json

```diff
@@ -3,36 +3,36 @@
         "path": "/projects/traversal/src/lib",
         "url": "git@github.com:guillotinaweb/angular-traversal.git",
         "https": "https://github.com/guillotinaweb/angular-traversal.git",
-        "branch": "master"
+        "tag": "1.2.6"
     },
     "grange-core": {
         "url": "git@github.com:guillotinaweb/grange-core.git",
         "https": "https://github.com/guillotinaweb/grange-core.git",
         "path": "/projects/grange-core/src",
-        "branch": "master"
+        "branch": "wip_login"
```

Comment:
put the proper grange-core version once the other PR is merged

Reply:
Its still WIP, so yes